robot/EDAP/resources/EDA.py

```diff
@@ -197,74 +197,74 @@ def fill_address_form(self, **kwargs):
 
     def Verify_details_address(self, field, value):
```

Comment:
While we're in here, can we make this `snake_case` @dashattuck?

Reply:
Actually, never mind. Let's not muddy the water.

Reply:
I modified two defs - both were using Verify_